backend/modules/simulation/handlers/chat_handler.py

```
@@ -0,0 +1,678 @@
+                persona_id_str = mention_match.group(1)
+                
+                if persona_id_str.lower() == 'all':
+                    # Handle @all
+                    all_result = await self.handle_all_mention(
+                        orchestrator, message, current_scene, correct_scene_id
+                    )
+                    ai_response = all_result['ai_response']
+                    persona_name = all_result['persona_name']
+                    persona_id = all_result['persona_id']
+                    all_responses = all_result.get('responses', [])
+                    scene_personas_count = all_result.get('personas_count', 0)
```

Ruff (0.14.8)

Local variable `scene_personas_count` is assigned to but never used

Remove assignment to unused variable `scene_personas_count`

(F841)